docs/uri-formattering.md

```diff
@@ -0,0 +1,61 @@
+
+- https://api.bag.kadaster.nl/esd/huidigebevragingen/v1
+- https://api.brk.kadaster.nl/esd-eto-apikey/bevragen/v1/
+- https://api.kadaster.nl/lvwoz-eto/huidigebevragingen
+
+Wanneer er BFFs in het systeem voorkomen is het zelfs nodig om de urls meerdere keren te herschrijven.
+
+Onderstaand figuur 1 visualiseert voor een Single Page Applicatie en een Desktop Applicatie de weg die een request naar en een response van de BRP API maakt. Bij elke tussenstap moeten de urls worden aangepast.
+
+![client API communicatie zonder uri-format negotiation](img/1-communicatie-client-api-zonder-uri-negotiation.svg)
+
+De Haal Centraal APIs maken gebruik van [URI Template](https://datatracker.ietf.org/doc/html/rfc6570) om urls te parameteriseren. Hiermee kunnen gelijksoortige urls (urls die alleen verschillen door path parameter waarde) worden gerepresenteerd met één URI template.
+
+Een URI template bevat één of meerdere expressions waarmee een verzameling van gelijksoortige URIs kan worden beschreven. Een expression is de tekst tussen een open accolade '{' en een sluit accolade '}' inclusief de accolade tekens. Er zijn meerdere expression types met bijbehorende expansie methode. Zie [Expressions](https://datatracker.ietf.org/doc/html/rfc6570#section-2.2).
+De Haal Centraal APIs gebruiken 'no operator expressions' zodat consumers 'Simple String Expansion'
+kunnen toepassen. Dit houdt in dat een {expression} zonder toepassing van logica kan worden vervangen met een waarde.
+Binnen de Haal Centraal APIs worden property namen gebruikt voor expressions. Een property naam is:
+- de naam van een property van een resource, v.b. burgerservicenummer of
+- de naam van een property van een gegevensgroep van een resource v.b. kinderen.burgerservicenummer
+
+Om de logica voor het aanpassen van Haal Centraal urls door Backend for Frontends en API Gateways zo simpel mogelijk te houden, en om de Haal Centraal API clients de mogelijkheid te bieden om hun gewenste uri formaat aan te kunnen geven, kunnen de Haal Centraal APIs gebruik maken van
+
+- [server-driven negotiation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Content_negotiation)
+- uri formats voor [Resource Identifiers](https://json-schema.org/draft/2020-12/json-schema-validation.html#rfc.section.7.3.5)
+- [uri-template](https://json-schema.org/draft/2020-12/json-schema-validation.html#rfc.section.7.3.6) 
+
+om de urls in een response te formatteren.
+
+Door middel van een custom HTTP header 'Accept-Uri-Format' kan een Haal Centraal API consumer in een request aangeven in welke format hij de urls in de response wil ontvangen.
+
+De volgende uri formats worden ondersteund:
```

Comment:
Maak de 3 mogelijke waarden (uri, uri-reference, uri-template) vet dat maakt n.m.m. in 1 oogoplsag duidelijker welke waarden gebruikt kunnen worden.